crates/solidity/inputs/language/src/definition.rs

```diff
@@ -1,21 +1,891 @@
+use std::cell::OnceCell;
+use std::collections::BTreeSet;
+use std::collections::HashMap;
+use std::rc::Rc;
+
+use codegen_grammar::GrammarElement;
+use codegen_grammar::ParserDefinitionNode;
+use codegen_grammar::PrecedenceOperatorModel;
+use codegen_grammar::PrecedenceParserDefinitionNode;
+use codegen_grammar::ScannerDefinitionNode;
+use codegen_grammar::VersionQuality;
+use codegen_grammar::VersionQualityRange;
+use codegen_language_definition::model::FieldsErrorRecovery;
+use codegen_language_definition::model::IndexMap;
+use codegen_language_definition::model::VersionSpecifier;
+use codegen_language_definition::model::{self, Identifier, Item};
+
 pub use solidity::SolidityDefinition;
 
+pub trait GrammarConstructorDslV2 {
```

Comment:
nit: if these are utils to convert to DSL v1, I suggest moving it to a different file, keeping only the definition here, to make it easier to work with the already existing 6K+ lines of code.

Reply:
Moved in 0e0d58e807e37000046f99bdeb75e7e627fa953c